airflow/providers/amazon/aws/operators/eventbridge.py

```diff
@@ -85,3 +85,77 @@ def execute(self, context: Context):
 
         if self.do_xcom_push:
             return [e["EventId"] for e in response["Entries"]]
+
+
+class EventBridgePutRuleOperator(BaseOperator):
+    """
+    Create or update a specified EventBridge rule.
+
```

Comment:
For documentation purposes, please link the operator with the documentation. Could you please do it for `EventBridgePutEventsOperator` as well?

```suggestion

    .. seealso::
        For more information on how to use this operator, take a look at the guide:
        :ref:`howto/operator: EventBridgePutRuleOperator`

```